npm install errors

When I try to install `@react-native-community/datetimepicker` package after installing `react-native-ui-lib`, am getting following error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-native-ui-lib@5.29.0
npm ERR! Found: @react-native-community/datetimepicker@3.5.2
npm ERR! node_modules/@react-native-community/datetimepicker
npm ERR!   @react-native-community/datetimepicker@"3.5.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @react-native-community/datetimepicker@"^2.1.0" from react-native-ui-lib@5.29.0
npm ERR! node_modules/react-native-ui-lib
npm ERR!   react-native-ui-lib@"^5.29.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @react-native-community/datetimepicker@2.6.2
npm ERR! node_modules/@react-native-community/datetimepicker
npm ERR!   peer @react-native-community/datetimepicker@"^2.1.0" from react-native-ui-lib@5.29.0
npm ERR!   node_modules/react-native-ui-lib
npm ERR!     react-native-ui-lib@"^5.29.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

I also tried installing `@react-native-community/datetimepicker` first followed by `react-native-ui-lib` then I get below error:
```
npm ERR! Found: @react-native-community/datetimepicker@3.5.2
npm ERR! node_modules/@react-native-community/datetimepicker
npm ERR!   @react-native-community/datetimepicker@"^3.5.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @react-native-community/datetimepicker@"^2.1.0" from react-native-ui-lib@5.30.0
npm ERR! node_modules/react-native-ui-lib
npm ERR!   react-native-ui-lib@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Is this an issue with `react-native-ui-lib`?

--- a/package.json
+++ b/package.json
@@ -119,11 +119,7 @@
     "react": ">=17.0.1",
     "react-native": ">=0.64.1",
     "react-native-gesture-handler": ">=1.9.0",
-    "react-native-reanimated": ">=2.0.0",
-    "@react-native-community/blur": ">=3.4.1",
-    "@react-native-community/datetimepicker": "^2.1.0",
-    "@react-native-community/netinfo": "^5.6.2",
-    "@react-native-picker/picker": "^1.9.4"
+    "react-native-reanimated": ">=2.0.0"
   },
   "jest": {
     "preset": "react-native",

